main.js

```diff
@@ -0,0 +1,26 @@
+const menuOver = document.querySelector('#menu-over');
```

Comment:
- [ ] You have done a good job with the mobile menu :+1: but there is an issue because it is shown when I visit the page. I mean when your page loads the mobile menu is already open. Please, kindly fix this issue for a better user experience :wink: :pushpin: